components/application-broker/cmd/broker/main.go

```diff
@@ -136,7 +142,7 @@ func SetupServerAndRunControllers(cfg *config.Config, log *logrus.Entry, stopCh
 	srv := broker.New(sFact.Application(), sFact.Instance(), sFact.InstanceOperation(), accessChecker,
 		mClient.ApplicationconnectorV1alpha1(), siFacade,
 		mInformersGroup.ApplicationMappings().Lister(), brokerService,
-		&mClient, knClient, log, livenessCheckStatus)
+		&mClient, knClient, istioClient, log, livenessCheckStatus)
```

Comment:
In order to be consistent with mClient, usage of a pointer for istioversioned.Interface makes sense for me.

Reply:
@sayanh But the already used `knClient` for example is not a pointer. 🤔